README.md

```diff
@@ -10,17 +10,21 @@ You need python 3.7+, poetry, and some local configuration.  If you are using `b
 
 If you don't have `bash` and `make`, or if this didn't work, then the manual steps are listed below.
 
+### python conflict
```

Comment:
This is a shame! I thought we could use up to 3.9 at least 😞 

For the record, after this module we won't use these Ansible hosts again and Docker will be much easier to use with a newer version of Python, so these can always be restored later!

Reply:
Nice.  I'll look at reinstating the changes during module 5